src/OrchardCore.Cms.Web/appsettings.json

```diff
@@ -80,6 +80,9 @@
     //  "ChangePasswordUrl": "ChangePassword",
     //  "ExternalLoginsUrl": "ExternalLogins"
     //},
+    "OrchardCore_Elastic": {
```

Comment:
To be removed before merge

Reply:
But needs to be documented somewhere.

Reply:
Also is there a notion of tenant, or store, such that it could automatically use the tenant name to create an isolated for each tenant, on the same ES deployment?

Reply:
@sebastienros  Notion of Tenants Yes: https://github.com/OrchardCMS/OrchardCore/pull/9108/commits/3f946a4bb62ca0f46616d2a6e9f91f1a7c9c483d

Reply:
@sebastienros please resolve this, if it answers the question!